### ISSUE TITLE
Errors with Unity using C#6 (e.g. 2018.2.17f1)

### ISSUE THREAD

**Issue by FireDragonGameStudio** [Lymdun/MirrorConverter#14]
Unfortunately projects, which are not C#7 enabled, will not be convertable due to:

```Assets/Mirror/Migration/Editor/Components.cs(125,90): error CS1644: Feature `out variable declaration' cannot be used because it is not part of the C# 6.0 language specification```

```Assets/Mirror/Migration/Editor/Windows.cs(14,57): error CS1644: Feature `out variable declaration' cannot be used because it is not part of the C# 6.0 language specification```

```Assets/Mirror/Migration/Editor/Windows.cs(32,62): error CS1644: Feature `out variable declaration' cannot be used because it is not part of the C# 6.0 language specification```

Best regards

**Comment by FireDragonGameStudio** [comment]
I know, it can be easily solved by creating the variables, but it's imho inconvenient. Maybe doing it the old-fashioned way ;) will be a good solution, even though it's not that beautiful.

```csharp
...
int netComponentObsolete;
Components.FindAndReplaceUnetComponents(out netComponentObsolete);
...
```

**Comment by Lymdun** [comment]
My very first "won't fix", since Mirror specifies 2018.3.6+ now :D
